openlibrary/catalog/marc/parse.py

```diff
@@ -357,20 +378,32 @@ def read_publisher(rec):
     return edition
 
 
-def read_author_person(f):
-    f.remove_brackets()
+def name_from_list(name_parts: list[str]) -> str:
+    STRIP_CHARS = r' /,;:[]'
+    name = ' '.join(strip_foc(s).strip(STRIP_CHARS) for s in name_parts)
+    return remove_trailing_dot(name)
+
+
+def read_author_person(f, tag='100') -> dict:
+    """
+    This take either a MARC 100 Main Entry - Personal Name (non-repeatable) field
+      or
+    700 Added Entry - Personal Name (repeatable)
+      or
+    720 Added Entry - Uncontrolled Name (repeatable)
+    and returns an author import dict.
+    """
     author = {}
-    contents = f.get_contents(['a', 'b', 'c', 'd', 'e'])
+    contents = f.get_contents(['a', 'b', 'c', 'd', 'e', '6'])
     if 'a' not in contents and 'c' not in contents:
         return  # should at least be a name or title
```

Comment:
The function signature promises that we will return a dict.
```suggestion
        return  {}  # should at least be a name or title
```